editoast/editoast_schemas/src/train_schedule/train_schedule_options.rs

```diff
@@ -14,6 +14,8 @@ pub struct TrainScheduleOptions {
     #[derivative(Default(value = "true"))]
     #[serde(default = "default_use_electrical_profiles")]
     use_electrical_profiles: bool,
+    #[serde(default)]
+    ignore_infra_speed_limits: bool,
```

Comment:
Does this flag also ignore speed limits for the pathfinding (or just for simulation)?
If so we should rename it to something like: `ignore_speed_limits_during_simulation`.

Reply:
It's only for the simulation, i'll rename the editoast side, core side isn't being used in pathfinding